README.md

```diff
@@ -63,6 +63,58 @@ async fn main() {
 The SDK uses tokio's [`tracing`](https://docs.rs/tracing/latest/tracing/) crate to generate logs.
 Just configure it as usual through [`tracing_subscriber`](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/) to get your logs.
 
+### Testing
+
+The SDK uses [Testcontainers](https://rust.testcontainers.org/) to support integration testing using a Docker-deployed restate server.
+The `test-env` crate provides a framework for initializing the test environment, and an integration test example in `test-env/tests/test_container.rs`.
+
+```rust
+#[tokio::test]
+async fn test_container() {
+    tracing_subscriber::fmt::fmt()
+        .with_max_level(tracing::Level::INFO) // Set the maximum log level
+        .init();
```

Comment:
Rather than this, I would show the https://docs.rs/tracing-test/latest/tracing_test/ macro.

Reply:
I gave this a shot but the filtering mechanism is kind of a deal breaker for integration tests. I couldn't figure out a way around removing all filters vs only seeing the test-env logs, and looks like it's an open issue:

https://github.com/dbrgn/tracing-test/issues/22

Let me know if you know of a solution and I'll switch it to use the macro.

Reply:
I'm sorry i pointed you to the wrong crate!!!

This is the macro I meant that we use around restate: https://docs.rs/test-log/latest/test_log/attr.test.html
Anyway this is fine, let's leave it as it is, maybe it's more explicit and people can use this macro if they want to!